src/app/lib/config/services/igbo.js

```diff
@@ -85,7 +85,7 @@ const igbo = {
         rejectUrl: 'https://www.bbc.co.uk/usingthebbc/your-data-matters',
       },
       cookie: {
-        title: 'Ka anyị mara ma i kwernyere na iwu ndị a',
+        title: 'Ka anyị mara ma i kwenyere n’iwu ndị a',
```

Comment:
```suggestion
        title: "Ka anyị mara ma i kwenyere n'iwu ndị a",
```
to match the apostrophe in the reject cookies field